tests/test_device.py

```diff
@@ -436,19 +437,19 @@ def circuit_gauss(mag_alpha, phase_alpha, phi):
         node_gauss = qml.QNode(circuit_gauss, dev_gauss)
         num_evals_gauss = 12
 
-        for i in range(num_evals_gauss):
+        for _ in range(num_evals_gauss):
             node_gauss(0.015, 0.02, 0.005)
         assert dev_gauss.num_executions == num_evals_gauss
 
     wires_to_try = [
-        (1, Wires([0]), Wires([0])),
-        (4, Wires([1, 3]), Wires([1, 3])),
-        (["a", 2], Wires([2]), Wires([1])),
-        (["a", 2], Wires([2, "a"]), Wires([1, 0])),
+        (1, Wires([0])),
+        (4, Wires([1, 3])),
+        (["a", 2], Wires([2])),
+        (["a", 2], Wires([2, "a"])),
     ]
 
-    @pytest.mark.parametrize("dev_wires, wires_to_map, res", wires_to_try)
-    def test_map_wires_caches(self, dev_wires, wires_to_map, res, mock_device):
+    @pytest.mark.parametrize("dev_wires, wires_to_map", wires_to_try)
+    def test_map_wires_caches(self, dev_wires, wires_to_map, mock_device):
```

Comment:
was the last argument being ignored here?

Reply:
Exactly, it simply is not used